packages/shared/src/domain/gender.ts

```diff
@@ -16,6 +18,8 @@ export function normalizeGenderSafe(gender: string): "F" | "M" | "O" | "U" | und
 
 export function normalizeGender(gender: string): GenderAtBirth {
   const genderOrUndefined = normalizeGenderSafe(gender);
-  if (!genderOrUndefined) throw new Error("Invalid gender");
+  if (!genderOrUndefined) {
+    throw new BadRequestError("Invalid gender", undefined, { gender });
```

Comment:
nit maybe this should be InvalidArgumentError since its a general purpose function?